contrib/templates/experimental/scala-job/databricks_template_schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "properties": {
+    "project_name": {
+      "type": "string",
+      "description": "Name of project. Do not add spaces",
+      "order": 1
+    },
+    "custom_workspace_host": {
+      "type": "string",
+      "default": "{{workspace_host}}",
+      "description": "Workspace host url",
+      "order": 2
+    },
+    "artifacts_dest_path": {
+      "type": "string",
+      "description": "Destination path in Databricks where the JAR and other artifacts will be stored You can use /Workspace or /Volumes in Dedicated clusters but only /Volumes in Standard clusters.",
+      "order": 3
+    },
+    "existing_cluster_id": {
+      "type": "string",
+      "description": "Enter the cluster id for an existing cluster or leave empty to create a new cluster",
```

Comment:
Could we avoid this question?
* We don't recommend using all-purpose clusters for production. For development, users are always free to use all-purpose and they can follow docs (or README.md guidance to do so)
* We want to bias toward a minimal number of questions in the templates